### PR TITLE
Vanilla set up

### PR DESCRIPTION
**Pull request by richmccartney**
#### What's this PR do?

Updates:
- Package.json to use the new dev dependancies based off vanilla framework v1.0
- Updates .gitignore to follow the same content as vanilla framework v1.0
- Adds vanilla framework v1.0 as a dependancy 
- Removes install_dependancies.sh
- Updates .scss-lint.yml
- Adds required gulp js files
- Adds updated run file
#### Where should the reviewer start?

Pull the branch down and run `npm i`
#### How should this be manually tested?

Run `gulp test` and it should use the new tools from vanilla framework v1
#### Any background context you want to provide?

_n/a_
#### What are the relevant tickets?
#8
#### Screenshots (if appropriate)

_n/a_
#### Questions:

_n/a_

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -1,75 +1,28 @@
...
+var gulp = require('gulp');
+var wrench = require('wrench');
 
-gulp.task('docs', function() {
-    return gulp.src('scss/**/*.scss')
-        .pipe(sassdoc({ 'dest': 'build/docs'}));
+/*
+  concatenate all *.js / *.coffee files in the 'gulp' folder
+ */
+wrench.readdirSyncRecursive('./gulp').filter(function(file) {
```
Comment (anthonydillon) [inline gulpfile.js]: So we need this as we have no js in the theme?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Contributing to Vanilla Dashboard Theme
+
+So, you'd like to contribute to the Vanilla Dashboard Theme? Great!
```
Comment (anthonydillon) [inline CONTRIBUTING.md]: It should be `Dashboard Vanilla Theme`.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+
+## Bugs and issues
+
+We use the [Issues tracker](https://github.com/ubuntudesign/vanilla-dashboard-theme/issues) to track all our bugs and feature requests, collectively known as 'issues'.
```
Comment (anthonydillon) [inline CONTRIBUTING.md]: The github report should be `dashboard-vanilla-theme`

File: package.json [hunk 4]
```diff
@@ -1,55 +1,64 @@
 {
+  "name": "maas-gui-vanilla-theme",
```
Comment (anthonydillon) [inline package.json:2]: This should be dashboard-vanilla-theme.

File: package.json [hunk 5]
```diff
@@ -1,55 +1,64 @@
...
   "bugs": {
-    "url": "https://github.com/ubuntudesign/maas-gui-vanilla-theme/issues",
-    "email": "webteam@canonical.com"
-  },
-  "scripts": {
-    "clean": "rm -r build; rm -r node_modules"
+    "url": "https://github.com/ubuntudesign/maas-gui-vanilla-theme/issues"
   },
-  "version": "0.0.1"
+  "homepage": "https://github.com/ubuntudesign/maas-gui-vanilla-theme#readme"
```
Comment (anthonydillon) [inline package.json]: This package.json has two homepage values.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Thanks @richmccartney some comments inline.

File: gulpfile.js (hunk 1 above)
Comment (richmccartney) [reply gulpfile.js]: I can see this theme having JS in the future. Especially if there are react / angular components from the respective products. For the time being I will remove this.

**Comment by richmccartney** [comment]
@anthonydillon if you can give this a second review, the package.json changes wont work until the theme name has change.

**Review by anthonydillon — APPROVED** [review]
Awesome, thanks :+1:

**Comment by anthonydillon** [comment]
Can you please change naming back to `maas-gui-vanilla-theme`

**Pull request merged**